amxmodx/CLang.cpp

```diff
@@ -24,24 +23,29 @@ CVector<FILE *> FileList;
 #define INSERT_NEWLINE		4
 
 template<>
-int Compare<String>(const String &k1, const String &k2)
+int Compare<ke::AString>(const ke::AString &k1, const ke::AString &k2)
 {
-	return k1.compare(k2.c_str());
+	return k1.compare(k2.chars());
```

Comment:
There's an overload for ke::AString so `k1.compare(k2)` does the same thing. There are other places where  this should be used for consistency.